sklearn/preprocessing/_function_transformer.py

```diff
@@ -64,11 +64,31 @@ def fit(self, X, y=None):
             check_array(X, self.accept_sparse)
         return self
 
+    def get_feature_names(self, input_features=None):
+        """
+        Return feature names for output features
+
+        Parameters
+        ----------
+        input_features : list of string, length len(input_features), optional
+            String names for input features if available. By default,
+            None is used.
+
+        Returns
+        -------
+        output_feature_names : list of string, length len(input_features)
+
+        """
+        if input_features is not None:
```

Comment:
No, something like `FunctionTransformer(lambda X_df: X_df['col_a', 'col_b'])` should have feature names ['col_a', 'col_b'].